spec/seafoam/cfg/cfg_parser_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'seafoam'
+
+require 'rspec'
+
+require_relative '../spec_helpers'
+
+describe Seafoam::CFG::CFGParser do
+  before :all do
+    @example_cfg = File.expand_path('../../../examples/java/exampleIf.cfg', __dir__)
+  end
+
+  it 'can skip to an nmethod in a file' do
+    parser = Seafoam::CFG::CFGParser.new(@example_cfg)
+    parser.skip_over_cfg 'After code installation'
+    parser.read_nmethod
```

Comment:
Are there actually any expectations here? If we're just checking it works without an exception, we can `expect` that the result of this method is of the right class or something.